src/System.Reflection.Metadata/src/System/Reflection/Metadata/Ecma335/Encoding/BlobEncoders.cs

```diff
@@ -74,24 +75,21 @@ public MethodSignatureEncoder PropertySignature(bool isInstanceProperty = false)
             return new MethodSignatureEncoder(Builder, hasVarArgs: false);
         }
 
-        public void CustomAttributeSignature(out FixedArgumentsEncoder fixedArguments, out CustomAttributeNamedArgumentsEncoder namedArguments)
+        [return: TupleElementNames(new[] { "FixedArguments", "NamedArguments" })]
```

Comment:
Yes, changes are expected. This will likely ship with Dev15, so I hope we'll switch to real tuple syntax at some point before Dev15 RTM.